src/main/java/com/minecolonies/coremod/client/gui/WindowHutDeliveryman.java

```diff
@@ -1,31 +0,0 @@
-package com.minecolonies.coremod.client.gui;
```

Comment:
Basically, its `WindowHutWorkerPlaceholder`. There wasn't really anything in this class, so I'm fine with removing it for now.